docs/sdk-and-tools/indices/blocks.md

```diff
@@ -0,0 +1,93 @@
+---
+id: blocks
+title: blocks
+---
+
+The `_id` field for this index is composed of block hash hex encoded
+
+| Field                 | Description                                                                                                |
+|-----------------------|------------------------------------------------------------------------------------------------------------|
+| nonce                 | The nonce field represents sequence number of block.                                                       |
+| round                 | The round field represents the round when block was proposed and executed.                                 |
+| epoch                 | The epoch field represent the epoch when the block was proposed and executed.                              |
+| miniBlocksHashes      | The miniBlocksHashes field represents the hashed hex encoded of the miniblocks that was included in block. |
+| miniBlocksDetails     | The miniBlocksDetails field represents the details of all the miniblocks that was included in block.       |
```

Comment:
The miniBlocksDetails field contains an array of structures indicating processing details of the miniblocks, such as the index of the first processed transaction